4.classify-features/classify-features.py

```diff
@@ -16,34 +16,48 @@
 classification_utils = importlib.import_module("classification-utils")
 
 
-# ### Download/load `phenotypic_profiling` model
+# ### Download/load `phenotypic_profiling` models
 
 # In[2]:
 
 
-gh_hash = "64cfc46ecd92f1956af199c81f8ecf4dc292718f"
-file_url = f"https://raw.github.com/WayScience/phenotypic_profiling_model/{gh_hash}/2.train_model/models/log_reg_model.joblib"
-log_reg_model_path = pathlib.Path("log_reg_model.joblib")
-urllib.request.urlretrieve(file_url, log_reg_model_path)
+gh_hash = "44e2741058c4d38edc137dc2caf5ea1f94b02410"
+final_model_file_url = f"https://raw.github.com/WayScience/phenotypic_profiling_model/{gh_hash}/2.train_model/models/log_reg_model.joblib"
+shuffled_baseline_file_url = f"https://raw.github.com/WayScience/phenotypic_profiling_model/{gh_hash}/2.train_model/models/shuffled_baseline_log_reg_model.joblib"
+
+models_path = pathlib.Path("phenotypic_profiling_models/")
+models_path.mkdir(exist_ok=True, parents=True)
+
+log_reg_model_path = pathlib.Path(f"{models_path}/log_reg_model.joblib")
+urllib.request.urlretrieve(final_model_file_url, log_reg_model_path)
 log_reg_model = joblib.load(log_reg_model_path)
 
+shuffled_baseline_model_path = pathlib.Path(f"{models_path}/shuffled_baseline_log_reg_model.joblib")
+urllib.request.urlretrieve(shuffled_baseline_file_url, shuffled_baseline_model_path)
+shuffled_baseline_model = joblib.load(shuffled_baseline_model_path)
```

Comment:
For these lines (and the earlier ones which are similar), consider using a file object directly with `joblib.load` to perform the read more directly when allowed. For example:
```suggestion
shuffled_baseline_model = joblib.load(
    urllib.request.urlopen(shuffled_baseline_file_url)
)
```

Reply:
I think in this case it makes sense to use `urllib.request.urlretrieve` to save also save the phenotypic_profiling model. This way, the model is preserved in this repository, which might be useful for later steps in the pipeline.